source/stormwater.rst

```diff
@@ -0,0 +1,329 @@
+
+    - **RR_mean**: Average retention ratio over this polygon
+
+    - **RV_sum**: Total retention volume over this polygon in :math:`m^3/yr`
+
+    - **IR_mean** (if infiltration data provided): Average infiltration ratio over this polygon
+
+    - **IV_sum** (if infiltration data provided): Total infiltration volume over this polygon in :math:`m^3/yr`
+
+    - **avoided_p** (for each pollutant :math:`p`): Total avoided amount of pollutant over this polygon in :math:`kg/yr`
+
+    - **load_p** (for each pollutant :math:`p`): Total amount of pollutant in runoff over this polygon in :math:`kg/yr`
+
+    - **val_sum** (if value data provided): Total value of the retained volume of water over this polygon in :math:`currency/yr`
+
+Intermediate Outputs
+^^^^^^^^^^^^^^^^^^^^
+
+- **lulc_aligned.tif**: Copy of the soil group raster input, cropped to the intersection of the three raster inputs
+
+- **soil_group_aligned.tif**: Copy of the soil group raster input, aligned to the LULC raster and cropped to the intersection of the three raster inputs
+
+- **precipitation_aligned.tif**: Copy of the precipitation raster input, aligned to the LULC raster and cropped to the intersection of the three raster inputs
+
+- **reprojected_centerlines.gpkg**: Copy of the road centerlines vector input, reprojected to the LULC raster projection
+
+- **rasterized_centerlines.tif**: A rasterized version of the reprojected centerlines vector, where 1 means the pixel is a road and 0 means it isn't
+
+- **is_impervious_lulc.tif**: A binary raster derived from the LULC raster and biophysical table `is_impervious` column, where 1 means the pixel has a directly-connected impervious LULC type, and 0 means it does not
+
+- **road_distance.tif**: A raster derived from the rasterized centerlines map, where each pixel's value is its minimum distance to a road pixel (measured centerpoint-to-centerpoint)
+
+- **impervious_lulc_distance.tif**: A raster derived from the is_impervious_lulc map, where each pixel's value is its minimum distance to an impervious LULC pixel (measured centerpoint-to-centerpoint)
+
+- **near_road.tif**: A binary raster derived from the road_distance map, where 1 means the pixel is within the retention radius of a road pixel, and 0 means it isn't
+
+- **near_impervious_lulc.tif**: A binary raster derived from the impervious_lulc_distance map, where 1 means the pixel is within the retention radius of an impervious LULC pixel, and 0 means it isn't
+
+- **search_kernel.tif**: A binary raster representing the search kernel that is convolved with the retention_ratio raster to calculate the averaged retention ratio within the retention radius of each pixel
+
+- **ratio_average.tif**: A raster where each pixel's value is the average of its neighborhood of pixels in the retention_ratio map, calculated by convolving the search kernel with the retention ratio raster
+
+
+.. _Input Guidance:
+
+Input Guidance
+==============
+
```

Comment:
Feedback from Milan:
> The retention radius parameter is completely user definable, but no ‘default’ value is given. I have some trouble finding an adequate value in literature for this. Could you maybe provide a default value for this setting, or a literature reference to learn more about a suitable value?

We should ideally provide some guidance on every input.